### ISSUE TITLE
[BUG] Stop Token Movement action not working for players

### ISSUE THREAD

**Issue by Bebetigno** [ironmonk88/monks-active-tiles#700]
**Describe the bug**
The "Stop Token Movement" action doesn't work when a player is moving the token that triggers it. It does work as intended when the GM is moving the token instead.

**To Reproduce**
I tested this on a clean scene, no other modules loaded than Monk's Active Tile Triggers. System is PF2e.
![image](https://github.com/ironmonk88/monks-active-tiles/assets/157961989/94bbf27a-b10b-467a-82ad-0f170faf2ab1)

Create an active tile set to trigger on "Enter", by anyone and by all tokens.
![image](https://github.com/ironmonk88/monks-active-tiles/assets/157961989/d672cc7d-6d10-40eb-a330-7ef39035a973)

Add two actions: one to create a "Chat Message" (to check if the tile actually triggered), the other to "Stop Token Movement".
![image](https://github.com/ironmonk88/monks-active-tiles/assets/157961989/837f4b3c-476b-4081-a041-8975a5bfd14a)

Put a player's token on the canvas. Move the token across the tile: the result is that, as expected, the token stops its movement in the triggered tile.
![image](https://github.com/ironmonk88/monks-active-tiles/assets/157961989/b2c36b37-4c30-42cb-8caa-51077797cfe6)
![image](https://github.com/ironmonk88/monks-active-tiles/assets/157961989/6aaa0eb4-e790-4d05-92a5-8b112324c011)
![image](https://github.com/ironmonk88/monks-active-tiles/assets/157961989/c94c851c-486e-428b-827c-a4fc25bc7717)

Log in as the token's player and repeat the movement across the tile: this time the movement doesn't stop. The chat message shows that the tile was indeed triggered.
![image](https://github.com/ironmonk88/monks-active-tiles/assets/157961989/1b812415-6d12-4804-9012-424d2c740064)
![image](https://github.com/ironmonk88/monks-active-tiles/assets/157961989/0c460777-98be-4911-88a3-a8aea5ca7815)
![image](https://github.com/ironmonk88/monks-active-tiles/assets/157961989/47bd1d87-e2cb-408f-ab6a-9e41f055e57d)

**Expected behavior**
The token's movement should be stopped by the tile when the player is moving it, too.

**Please complete as much of the following information as possible:**
 - Browser: Foundry application for GM, Chrome for player. The same behavior happens if I switch.
 - Foundry Version: v11.315
 - System: Pathfinder Second Edition 5.12.7
 - Module version: 11.21
 - Are there any errors in the console: no errors
 - Have you tried using `Find the Culprit` to make sure it isn't a module interaction: no other modules active

**Additional context**
I recently updated from v9 to v11 (along with system and modules); I can't remember this behavior on the previous version.

**Comment by Bebetigno** [comment]
I see that you are investigating the issue; I have asked around on Discord, and no one seems to have noticed this problem. It would be strange if it only happened to me though, so I'll add some information that may help you.
I made my test while running Foundry as a GM, and connecting as a player with Chrome, on the same PC; I also did a quick test (which confirmed the issue) connecting as the same player from a laptop (also with Chrome), but doing so is little awkward because I don't have hardware acceleration there and everything is very lagging and slow.
If it can help, I can post my game settings too.
